apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx

```diff
@@ -97,21 +97,21 @@ const NavigationBar = () => {
       <nav
         data-state={snap.navigationPanelOpen ? 'expanded' : 'collapsed'}
         className={cn(
-          'group py-2 z-10 h-full w-14 data-[state=expanded]:w-[13rem]',
+          'group z-10 h-full w-14 data-[state=expanded]:w-[13rem]',
           'border-r bg-studio border-default data-[state=expanded]:shadow-xl',
           'transition-width duration-200',
-          'hide-scrollbar flex flex-col justify-between overflow-y-auto'
+          'hide-scrollbar flex flex-col overflow-y-auto'
         )}
         onMouseEnter={() => snap.setNavigationPanelOpen(true)}
         onMouseLeave={() => {
           if (!userDropdownOpen) snap.setNavigationPanelOpen(false)
         }}
       >
-        <ul className="flex flex-col gap-y-1 justify-start px-2">
-          {(!navLayoutV2 || !IS_PLATFORM) && (
+        {(!navLayoutV2 || !IS_PLATFORM) && (
+          <div className="sticky top-0 mb-1 py-1 px-2 flex h-[48px] bg-studio z-50">
```

Comment:
z-50 is possibly wrong as z-50 and above is reserved for overlays (Dropdown menu, tooltips etc), so this nav could end up being rendered on top of overlays. z-10 is safe to use.

Reply:
Thanks @MildTomato, I will check.